docs/recipes/plugin/package.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "plugin-lighthouse-recipe",
+  "private": true,
+  "dependencies": {
+    "lighthouse": "^4.3.0"
```

Comment:
intentional as `--plugins` landed after 4.2.0 :)

Reply:
> intentional as --plugins landed after 4.2.0 :)

actually, jk, that's just really broken. Instead, now it just won't run the plugin since `--plugins` won't be recognized, but it won't throw an error. The release of 4.3.0 and semver (and/or we remember to update the recipe) will make it work when that comes out